src/app/views/login-router/login-router.coffee

```diff
@@ -0,0 +1,13 @@
+angular.module 'mnoEnterpriseAngular'
+  .controller 'LoginRouterCtrl', ($scope, MnoeCurrentUser, $window) ->
+    'ngInject'
+	
+    $scope.redirect = ->
+    MnoeCurrentUser.get().then(
```

Comment:
@alexnoox     
alexnoox added a note an hour ago
You have an indentation problem.

It results with $scope.redirect = -> being an empty function, doing nothing. But the code under it executed while the controller is loaded.

I think you should remove the empty function and use the regular controller init.